src/main/charts/crowd/Changelog.md

```diff
@@ -4,11 +4,12 @@
 
 **Release date:** TBD
 
-![AppVersion: 7.13.5](https://img.shields.io/static/v1?label=AppVersion&message=7.13.5&color=success&logo=)
```

Comment:
Was 7.13.5 mistake?

Reply:
yea, AppVersion should be the product version, so Crowd should have 5.0.0.